src/add/operator/zip-static.ts

```diff
@@ -1,5 +1,5 @@
 import {Observable} from '../../Observable';
 import {zip} from '../../operator/zip-static';
-Observable.zip = zip;
+Observable.zip = <any>zip;
```

Comment:
Yes, if dropping this change breaks the build we should modify the script to generate code that deals with this case. But I'm not sure if this is the case since this is the only static operator that required casting. The other change (`src/add/operator/zip.ts`) was only a renaming, so it can be dropped without problems.